recipes/r-prettycode/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = '1.0.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-prettycode
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/prettycode_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/prettycode/prettycode_{{ version }}.tar.gz
+  sha256: 4f1274a4cc683a16823b0989aea2803053dec3cbd5caeb6bba64e67d312ee77b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Comment:
This package doesn't need to be compiled, so it should be a noarch recipe. Please add the line `noarch: generic` to this section.

If you didn't delete this line, you probably need to update `conda-build` to use the latest version of the skeleton (I confirmed that version 3.18.2 creates a noarch recipe.

Reply:
Weird, I have `conda-build=3.18.2` and it doesn't create a `noarch` recipe for me.

Reply:
Ah, then you may need to update your local copy of the `conda_r_skeleton_helper`. It adds the flag `--use-noarch-generic`.

Reply:
I wasn't aware of `conda_r_skeleton_helper`, I was using plain `conda skeleton cran prettycode`. Pushed a new commit that uses this now.

Reply:
Wow, well then take that as a complement! Your recipe looked so well-formatted that I assumed you had used it :-)